src/network/ClientConnection.ts

```diff
@@ -486,12 +486,19 @@ export class ClientConnection {
         let message = this.toString() + ' closed. Reason: ';
         if (this.closedReason != null) {
             message += this.closedReason;
-        } else if (this.closedCause != null) {
-            message += this.closedCause.name + '[' + this.closedCause.message + ']';
         } else {
             message += 'Socket explicitly closed';
         }
 
+        if (this.closedCause != null) {
```

Comment:
Minor: If both `closedReason` and `closedCause` are not null, this is producing messages like `Reason: Failed to authenticate connection - Authentication failed. The configured cluster name..`. Maybe we could change those `connection.close` calls and make the produced log message better

Reply:
Good suggestion. Tried to improve log messages in c8d38f8e73898482b02f080482939adb1e759255

Please take another look.

Reply:
Looks good, thanks